### DISCUSSION TITLE
Query engine binary for current platform "windows" could not be found

### DISCUSSION THREAD

**Discussion by reflash** [prisma/prisma#4320] [Q&A] [answered]
`This probably happens, because you built Prisma Client on a different platform`
Is there a way to specify multiple platforms to build for? I'm deploying to a windows machine so it fails to find a build for windows.

**Comment by reflash** [comment, marked as answer]
Solved by specifying binary targets (https://www.prisma.io/docs/concepts/components/prisma-schema/generators#binary-targets-prisma-client-js). 
Avaliable options can be found here - https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#binarytargets-options

In my case it was:
```prisma
generator client {
  provider = "prisma-client-js"
  binaryTargets = ["native", "windows"]
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
